tldr.py

```diff
@@ -34,12 +34,15 @@
 USE_NETWORK = int(os.environ.get('TLDR_NETWORK_ENABLED', '1')) > 0
 USE_CACHE = int(os.environ.get('TLDR_CACHE_ENABLED', '1')) > 0
 MAX_CACHE_AGE = int(os.environ.get('TLDR_CACHE_MAX_AGE', 24*7))
+CAFILE = os.path.expanduser(os.environ.get('TLDR_CERT', None))
```

Comment:
```suggestion
CAFILE = None if os.environ.get('TLDR_CERT', None) is None else Path(os.environ.get('TLDR_CERT')).expanduser()
```

This change should fix the error when $TLDR_CERT is not set. Also, we prefer to use `pathlib` instead of `os.path`.